services/worker/tests/fixtures/files.py

```diff
@@ -64,3 +64,52 @@ def extra_fields_readme(tmp_path_factory: pytest.TempPathFactory) -> str:
     with open(path, "w", newline="") as f:
         f.writelines(f"{line}\n" for line in lines)
     return path
+
+
+DATASET_SCRIPT_WITH_EXTERNAL_FILES_CONTENT = """
+import datasets
+
+_URLS = {
+    "train": [
+        "https://huggingface.co/datasets/lhoestq/test/resolve/main/some_text.txt",
```

Comment:
This adds a dependency on the prod Hub for the tests. Why not, we didn't have (for the workers) as for now.

We could generate the repo on the hub-ci, depend on it, and then delete it. Maybe it's too much?

Reply:
It's simpler to use this one directly. And I already use it elsewhere for tests so it's not going to change.